app/controllers/users/registrations_controller.rb

```diff
@@ -90,13 +109,6 @@ def edit
   # Iter 1-2
   def update
     super
-    # user = User.find_by_email(params[:user][:email])
```

Comment:
Are these unnecessary or redone else where?

Reply:
seems that the previous team was not using the code and was calling a super method so I am assuming that it is done in the parent update